recipes/aws-sdk-cpp/run_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Stop on first error
+set -e
+
+if [ "$(uname)" == "Darwin" ]; then
+    # OSX
+    clang++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib" -Wl,-rpath,"${PREFIX}/lib" -laws-cpp-sdk-core -o test-core
+else
+    # Linux
+    g++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib" -Wl,-rpath,"${PREFIX}/lib" -laws-cpp-sdk-core -o test-core
+fi
```

Comment:
Why did you remove running the executable?

Reply:
sorry, just trying to wrap this up quickly before leaving work

added it back